Backend/CarRentalApp/CarRentalApp/Services/Authentication/RoleService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using CarRentalApp.Models.DTOs;
+using CarRentalApp.Services.Identity;
+
+namespace CarRentalApp.Services.Authentication;
+
+public class RoleService
+{
+    private readonly UserService _userService;
+
+    public RoleService(UserService userService)
+    {
+        _userService = userService;
+    }
+
+    public async Task UpgradeRoleAsync(UserDTO userDTO)
+    {
+        var user = await _userService.GetExistingUserAsync(userDTO);
+
+        await _userService.UpgradeRoleAsync(user);
```

Comment:
What is the purpose of this class? You just calling two methods of userService, why do you need separate class for that?